docs/1.7/obb.cmn.md

```diff
@@ -1,6 +1,31 @@
 # Oriented Bounding Box (OBB)
 
-Definition of an oriented bounding box.
+An Oriented Bounding Box (OBB) is a compact bounding volume representation, tightly fitting the geometries it represents. An OBBs' invariance to translation and rotation, makes it ideal as the optimal and default bounding volume representation in I3S.
+
+When constructing an OBB for I3S use, there are two considerations an implementer needs to be make based on the Coordinate Reference System (CRS) of the layer:
+
+## Constructing OBB for a global scene layer
+
+For an I3S layer in WGS84 geographic coordinate system, referred to as a *global scene layer* and identified by the wkid value of 4326 (see [spatial reference](/docs/1.7/spatialReference.cmn.md) in [3DSceneLayer](/docs/1.7/3DSceneLayer.cmn.md) resource): 
+docs/1.7/3DSceneLayer.cmn.md
+- the OBB should be constructed in an earth-centered, earth-fixed (ECEF) coordinate system equivalent to the layers' CRS.
+- the center component of the OBB is specified as longitude, latitude (in decimal degrees) and the elevation (z) in meters.
+- the halfSize component of the OBB is specified in meters.
+- the quaternion component is in reference to an ECEF coordinate system.
+
+When constructing an OBB for a global scene layer, the following steps must occur:

```

Comment:
This might need to be
```
When constructing an OBB for a global scene layer, the following steps must occur:
- Find center point, half sizes, and quaternion vector necessary for deriving your OBB in native CRS
- project center point to ECEF
- Compute OBB in ECEF using this point and previously found half sizes and quat
- Convert each corner and center point of derived OBB back to native CRS. 
```

Reply:
@kylemann16 thanks! We discussed about adding a detailed step by step procedure but decided against it here, as it would be better to do so in a separate obb construction section (sort of implementers' guide), which we plan to do.